### ISSUE TITLE
Full Compute node image

### ISSUE THREAD

**Issue by ElliotAOram** [ISISScientificComputing/autoreduce#331]
Issue raised by: [developer]

### What?
Before we can push ahead with cloud computing, we need to ensure we can reliably spin up a full compute node with all the requirements. Much of the work has been done, but we are still missing a working way to:
* Install Mantid
* Mount the Archive
* Deal with Mantid user properties
These all need to be investigated and resolved in this issue

### Where?
Ansible script 
Docker for Mantid?

### How to test the issue is resolved
* Spin up a cloud instance
* Image the cloud instance with the compute node image
* Validate that it is running as expected (Manually submission)

**Comment by ElliotAOram** [comment]
This is no longer relevant due to change to use mantid docker
